x/rollapp/types/key_rollapp.go

```diff
@@ -36,3 +37,15 @@ func RollappByEIP155Key(
 
 	return key
 }
+
+func RollappByAliasKey(
+	alias string,
+) []byte {
+	var key []byte
+
+	aliasBytes := []byte(alias)
+	key = append(key, aliasBytes...)
+	key = append(key, []byte("/")...)
```

Comment:
and what if alis contains `/`?

Reply:
fixed by allowing only alphanumeric characters and `_`